src/css/custom.css

```diff
@@ -128,4 +128,8 @@ article header h1 {
   .support-wrapper li {
     padding: 10px 15px;
   }
+}
+
+.text-center {
+  text-align: center;
 }
```

Comment:
@Arhell Please add extra line at the end of the file.

Reply:
Done